requirements/edx/base.txt

```diff
@@ -33,6 +33,7 @@ django-oauth-toolkit==0.10.0
 django-pipeline-forgiving==1.0.0
 django-pyfs==1.0.6
 django-sekizai==0.8.2
+django-classy-tags==0.9.0
```

Comment:
What version was installed by default? Do you replace the version with older one or newer one?
Can it possibly affect other dependant functionality?

Reply:
module version was picked from `pip freeze` of another ginkgo installation (which was not updated recently)

Reply:
@TamoshaytisV please resolve this thread )